### PR TITLE
{Network} Bump azure-mgmt-network from 19.3.0 to 20.0.0

### PR DESCRIPTION
**Pull request by necusjz**
**Related command**

**Description**

Bullet points:
- Removed "live_only" mentioned in: https://github.com/Azure/azure-cli/pull/22485;
- Fixed "content_type" introduced by 2017-10-01;
- Temporarily bypass several rdbms tests (@DaeunYim for awareness):
    - test_mysql_flexible_server_georestore_mgmt;
    - test_mysql_flexible_server_restore_mgmt;
    - test_mysql_flexible_server_vnet_mgmt_supplied_subnetid;
    - test_mysql_flexible_server_vnet_mgmt_supplied_subnet_id_in_different_rg;
    - test_mysql_flexible_server_vnet_mgmt_supplied_vname_and_subnetname;
    - test_postgres_flexible_server_vnet_mgmt_supplied_vname_and_subnetname;
    - test_postgres_flexible_server_restore_mgmt;
    - test_mysql_flexible_server_existing_private_dns_zone;
    - test_mysql_flexible_server_new_private_dns_zone;

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

**Review by calvinhzy — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Comment by jiasli** [comment]
I saw `azure_mgmt_network-20.0.0-py3-none-any.whl` is smaller than `azure_mgmt_network-19.3.0-py2.py3-none-any.whl`:

https://pypi.org/project/azure-mgmt-network/20.0.0/#files
image

https://pypi.org/project/azure-mgmt-network/19.3.0/#files
image

This is a good sign for package size!

File: src/azure-cli-core/azure/cli/core/commands/__init__.py [hunk 1]
```diff
@@ -130,6 +130,7 @@ def path(self, args, kwargs):
             raise CLIError('cloud name unexpectedly empty')
         copy_kwargs = kwargs.copy()
         copy_kwargs.pop('self', None)
+        copy_kwargs.pop('content_type', None)
```
Comment (jiasli) [inline src/azure-cli-core/azure/cli/core/commands/__init__.py:133]: Please leave a comment here.

**Review by evelyn-ys — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

File: src/azure-cli-core/azure/cli/core/commands/__init__.py (hunk 1 above)
Comment (jiasli) [reply src/azure-cli-core/azure/cli/core/commands/__init__.py:134]: This is for the signature change of Python SDK methods (https://github.com/Azure/azure-sdk-for-python/pull/24162):

https://github.com/Azure/azure-sdk-for-python/blob/83ff40e58aad2fbd3f912e2f302a342f1e89a1f9/sdk/network/azure-mgmt-network/azure/mgmt/network/v2017_10_01/operations/_operations.py#L22890-L22899

```py
    @distributed_trace
    def begin_create_or_update(
        self,
        resource_group_name: str,
        virtual_network_name: str,
        parameters: "_models.VirtualNetwork",
        *,
        content_type: Optional[str] = "application/json",
        **kwargs: Any
    ) -> LROPoller["_models.VirtualNetwork"]:
```

**Pull request merged**